packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.classNames.ts

```diff
@@ -27,7 +27,7 @@ export const getClassNames = memoizeFunction(
       root: mergeStyles(styles.root, className),
       labelWrapper: mergeStyles(styles.labelWrapper, _getStyleForLabelBasedOnPosition(labelPosition, styles)),
       icon: mergeStyles(styles.icon, disabled && styles.iconDisabled),
-      label: mergeStyles(styles.label, disabled && styles.labelDisabled),
```

Comment:
clarification question: why is styles.labelDisabled being deleted here if it's still in SpinButton.styles.ts?

Reply:
Because it’s an empty style area I am deprecating with PR and instead let `Label` component handle its disabled state styles by passing the `disabled` prop to it.